app/Http/Controllers/TaskController.php

```diff
@@ -45,7 +46,8 @@ public function store(Request $request)
      */
     public function show($id)
     {
-        //
+        $Tasks = User::find($id)->tasks;
+        return view('Users.Tasks',compact('Tasks'));
```

Comment:
```suggestion
        $tasks = User::find($id)->tasks;
        
        return view('users.tasks', compact('tasks'));
```
convention, chú ý cách đặt tên biến, tên file, convention